server/controllers/importers/sif.js

```diff
@@ -122,20 +122,23 @@ module.exports = function (sif) {
                 });
             }
         });
+  }
+
+  var network = {
+    genes: emptySifFile ? [] : genes.map(function (geneName) {
+      return { name: geneName };
+    }),
```

Comment:
Yes, it's just used for emptySifFile. I set sif = " " (for lack of something better to set it to so it doesn't throw errors in the parsing) when the importer detects that the file is completely empty. Without this, genes was returning [ " " ]. Sort of a workaround I'll see if I can come up with a cleaner way to do this.